pygmt/helpers/utils.py

```diff
@@ -128,10 +129,20 @@ def build_arg_string(kwargs):
 
     >>> print(
     ...     build_arg_string(
-    ...         dict(R="1/2/3/4", J="X4i", P="", E=200, X=None, Y=None)
+    ...         dict(
+    ...             R="1/2/3/4",
+    ...             J="X4i",
+    ...             P="",
+    ...             E=200,
+    ...             X=None,
+    ...             Y=None,
+    ...             A=True,
+    ...             B=False,
+    ...             Z=0,
```

Comment:
Sort alphabetically and use SI units :slightly_smiling_face:

```suggestion
    ...             A=True,
    ...             B=False,
    ...             E=200,
    ...             J="X4c",
    ...             P="",
    ...             R="1/2/3/4",
    ...             X=None,
    ...             Y=None,
    ...             Z=0,
```